tests/mpi_54rank/test_external_grid_1x1.py

```diff
@@ -0,0 +1,154 @@
+import math
+import os
+
+import numpy as np
+import xarray as xr
+import yaml
+
+import pace.util
+from pace.driver import Driver, DriverConfig
+from pace.util.constants import PI, RADIUS
+from pace.util.mpi import MPIComm
+
+
+DIR = os.path.dirname(os.path.abspath(__file__))
+
+
+def get_cube_comm(layout, comm: MPIComm):
+    return pace.util.CubedSphereCommunicator(
+        comm=comm,
+        partitioner=pace.util.CubedSpherePartitioner(
+            pace.util.TilePartitioner(layout=layout)
+        ),
+    )
+
+
+def get_tile_num(comm: MPIComm):
+    return pace.util.get_tile_index(comm.rank, comm.partitioner.total_ranks)
```

Comment:
If you use pace.util.get_tile_number here you don't have to add 1 later

Reply:
Ok will use that instead